App/CourseApp/SharedUI/UIConstants.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIConstants.swift
+//  CourseApp
+//
+//  Created by Marcel Mravec on 17.05.2024.
+//
+
+import Foundation
+
+enum UIConstants {
```

Comment:
I would suggest to split it and avoid global constants. If they are used globally let create specific enums for design system like rounding, scales, spacing etc. 
For those used on some screens I would keep them just there